ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -161,26 +171,53 @@ data Cmd blk it rdr
 -- something we are testing in 'prop_trace', see
 -- 'invalidBlockNeverValidatedAgain'.
 
+-- = Blocks from the future
```

Comment:
Instead: add a separate "advanceTime" function to the model that is called when executing `Add(Future)Block`.